### ISSUE TITLE
Screwed HTML code in the Orders screen

### ISSUE THREAD

**Issue by mrleemon** [decarvalhoaa/woo-poly-integration#3]
Hi,

I've found a non critical bug in the plugin. If you click the "n items" link of an order in the Orders screen, the HTML code is screwed.

![woopoly](https://cloud.githubusercontent.com/assets/2060659/15894557/cb0d5ab8-2d86-11e6-8c2b-2ea5de36132b.jpg)

The problem can be found in the `return sprintf()` line in the `translateProductNameInOrdersDetails()` function in the `src/Hyyan/WPI/Order.php` file:

```
public function translateProductNameInOrdersDetails($name, $item)
{
    $id = $item['item_meta']['_product_id'][0];
    $product = Utilities::getProductTranslationByID($id);
    if($product) {
        if (!$product->is_visible()) {
            return $product->post->post_title;
        } else {
            return sprintf('<a href="%s">%s</a>', get_permalink($product->id), $product->post->post_title);
        }
    } else {
        return $name;
    }
}
```

**Comment by decarvalhoaa** [comment]
Thanks for reporting.

**Comment by decarvalhoaa** [comment]
Fixed in v0.27.

https://github.com/decarvalhoaa/woo-poly-integration/releases/download/v0.27/woo-poly-integration-0.27.zip

**Comment by mrleemon** [comment]
Thanks!
